src/subclassers.ts

```diff
@@ -16,35 +18,51 @@ export function WithInit(
   };
 }
 
```

Comment:
- [ ] can you remove the unnecessary return types here too?

Reply:
done